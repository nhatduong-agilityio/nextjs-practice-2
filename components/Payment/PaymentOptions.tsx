import { SimpleGrid, Stack, Text, useRadioGroup } from '@chakra-ui/react'
import React, { useState } from 'react'
import { BankMock, DigitalWalletMock, MiniMarketMock, PaymentMock } from '@mock/dataMock'
import ListRadioBoxImage from '@components/Common/ListRadioBoxImage'

const PaymentOptions = () => {
  const [paymentOption, setPaymentOption] = useState('')

  const { getRadioProps, getRootProps } = useRadioGroup({
    defaultValue: paymentOption,
    onChange: setPaymentOption,
  })

  return (
    <SimpleGrid style={{ margin: 0 }} columns={[1, 4]} w='full' {...getRootProps()}>
      <Stack w='full'>
        <Text size={['16', '18']} variant='large'>
          Minimarket
        </Text>
        <ListRadioBoxImage items={MiniMarketMock} getRadioProps={getRadioProps} />
      </Stack>
      <Stack w='full'>
        <Text size={['16', '18']} variant='large'>
          Digital Wallet
        </Text>
        <ListRadioBoxImage items={DigitalWalletMock} getRadioProps={getRadioProps} />
      </Stack>
      <Stack w='full'>
        <Text size={['16', '18']} variant='large'>
          Bank International Payment
        </Text>
        <ListRadioBoxImage items={BankMock} getRadioProps={getRadioProps} />
      </Stack>
      <Stack w='full'>
        <Text size={['16', '18']} variant='large'>
          International Payment
        </Text>
        <ListRadioBoxImage items={PaymentMock} getRadioProps={getRadioProps} />
      </Stack>
    </SimpleGrid>
  )
}

export default PaymentOptions
