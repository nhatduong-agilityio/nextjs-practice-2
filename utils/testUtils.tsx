import { ChakraProvider } from '@chakra-ui/react'
import { AuthProvider } from '@store/providers/AuthProvider'
import { LoadingProvider } from '@store/providers/LoadingProvider'
import type { RenderOptions, RenderResult } from '@testing-library/react'
import { render } from '@testing-library/react'
import { customTheme } from '@themes/index'
import type { ReactElement } from 'react'

const AllTheProviders = ({ children }: { children: React.ReactNode }) => {
  return (
    <ChakraProvider theme={customTheme}>
      <LoadingProvider>
        <AuthProvider>{children}</AuthProvider>
      </LoadingProvider>
    </ChakraProvider>
  )
}

const customRender = (ui: ReactElement, options?: Omit<RenderOptions, 'wrapper'>): RenderResult =>
  render(ui, { wrapper: AllTheProviders, ...options })

export * from '@testing-library/react'
export { customRender as render }
