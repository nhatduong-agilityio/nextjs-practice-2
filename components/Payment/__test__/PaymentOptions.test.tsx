// Utils
import { render } from '@utils/testUtils'

// Components
import PaymentOptions from '../PaymentOptions'

describe('PaymentOptions render', () => {
  it('Should PaymentOptions match Snapshot', () => {
    const { container } = render(<PaymentOptions />)

    expect(container).toMatchSnapshot()
  })
})
