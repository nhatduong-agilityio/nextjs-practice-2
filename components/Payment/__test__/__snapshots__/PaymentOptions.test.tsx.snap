// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentOptions render Should PaymentOptions match Snapshot 1`] = `
<div>
  <div
    class="css-n2r5bk"
    role="radiogroup"
    style="margin: 0px;"
  >
    <div
      class="chakra-stack css-ppmla1"
    >
      <p
        class="chakra-text css-1jsp72x"
      >
        Minimarket
      </p>
      <div
        class="chakra-stack css-vedre6"
        style="margin: 0px;"
      >
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r1:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="alfaMart"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Alfa Mart"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Falfa-mart.png%3Falt%3Dmedia%26token%3D1c0b2cba-977f-402d-a457-132d103de31b&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Falfa-mart.png%3Falt%3Dmedia%26token%3D1c0b2cba-977f-402d-a457-132d103de31b&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Falfa-mart.png%3Falt%3Dmedia%26token%3D1c0b2cba-977f-402d-a457-132d103de31b&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r2:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="indoMaret"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Indo Maret"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Findomaret.png%3Falt%3Dmedia%26token%3D651463b6-7b3d-44ac-a038-28aff66b4fbd&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Findomaret.png%3Falt%3Dmedia%26token%3D651463b6-7b3d-44ac-a038-28aff66b4fbd&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Findomaret.png%3Falt%3Dmedia%26token%3D651463b6-7b3d-44ac-a038-28aff66b4fbd&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
      </div>
    </div>
    <div
      class="chakra-stack css-ppmla1"
    >
      <p
        class="chakra-text css-1jsp72x"
      >
        Digital Wallet
      </p>
      <div
        class="chakra-stack css-vedre6"
        style="margin: 0px;"
      >
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r3:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="goPay"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Go Pay"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-f4624.appspot.com%2Fo%2Fimages%252Fgopay.png%3Falt%3Dmedia%26token%3Df2138737-db02-4422-98ab-0fa3c0bfffad&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-f4624.appspot.com%2Fo%2Fimages%252Fgopay.png%3Falt%3Dmedia%26token%3Df2138737-db02-4422-98ab-0fa3c0bfffad&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-f4624.appspot.com%2Fo%2Fimages%252Fgopay.png%3Falt%3Dmedia%26token%3Df2138737-db02-4422-98ab-0fa3c0bfffad&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r4:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="ovo"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Ovo"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fovo.png%3Falt%3Dmedia%26token%3Dcf38fe11-dc6e-448a-bcd4-ec918a0535e4&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fovo.png%3Falt%3Dmedia%26token%3Dcf38fe11-dc6e-448a-bcd4-ec918a0535e4&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fovo.png%3Falt%3Dmedia%26token%3Dcf38fe11-dc6e-448a-bcd4-ec918a0535e4&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r5:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="linkAja"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Link Aja"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Flink-aja.png%3Falt%3Dmedia%26token%3D209d51fd-b7d3-412c-afb9-f92bbb896d33&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Flink-aja.png%3Falt%3Dmedia%26token%3D209d51fd-b7d3-412c-afb9-f92bbb896d33&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Flink-aja.png%3Falt%3Dmedia%26token%3D209d51fd-b7d3-412c-afb9-f92bbb896d33&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
      </div>
    </div>
    <div
      class="chakra-stack css-ppmla1"
    >
      <p
        class="chakra-text css-1jsp72x"
      >
        Bank International Payment
      </p>
      <div
        class="chakra-stack css-vedre6"
        style="margin: 0px;"
      >
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r6:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="bca"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Bca Bank"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbca-bank.png%3Falt%3Dmedia%26token%3Dfbadf89a-1723-4760-868b-faa407bd004a&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbca-bank.png%3Falt%3Dmedia%26token%3Dfbadf89a-1723-4760-868b-faa407bd004a&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbca-bank.png%3Falt%3Dmedia%26token%3Dfbadf89a-1723-4760-868b-faa407bd004a&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r7:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="mandiri"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Mandiri Bank"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fmandiri-bank.png%3Falt%3Dmedia%26token%3D7d9fd118-5cba-484b-a775-81372ffd92c8&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fmandiri-bank.png%3Falt%3Dmedia%26token%3D7d9fd118-5cba-484b-a775-81372ffd92c8&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fmandiri-bank.png%3Falt%3Dmedia%26token%3D7d9fd118-5cba-484b-a775-81372ffd92c8&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r8:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="bri"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Bri Bank"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbri-bank.png%3Falt%3Dmedia%26token%3D637eb850-660a-4d5c-ac4d-7db1844a8d87&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbri-bank.png%3Falt%3Dmedia%26token%3D637eb850-660a-4d5c-ac4d-7db1844a8d87&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fbri-bank.png%3Falt%3Dmedia%26token%3D637eb850-660a-4d5c-ac4d-7db1844a8d87&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
      </div>
    </div>
    <div
      class="chakra-stack css-ppmla1"
    >
      <p
        class="chakra-text css-1jsp72x"
      >
        International Payment
      </p>
      <div
        class="chakra-stack css-vedre6"
        style="margin: 0px;"
      >
        <div
          class="chakra-stack css-1hyr0v9"
        >
          <label
            class="css-4g6ai3"
          >
            <input
              hidden=""
              id="radio-:r9:"
              name="radio-:r0:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="radio"
              value="payPal"
            />
            <div
              aria-hidden="true"
              class="css-fnt4nb"
            />
          </label>
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 50px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
          >
            <img
              alt="Logo brand of Paypal"
              data-nimg="fixed"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fpaypal.png%3Falt%3Dmedia%26token%3D300e38ad-6415-44b4-b44e-095012e3fb9c&w=256&q=75"
              srcset="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fpaypal.png%3Falt%3Dmedia%26token%3D300e38ad-6415-44b4-b44e-095012e3fb9c&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fshop-bag-ffd59.appspot.com%2Fo%2Fimages%252Fpaypal.png%3Falt%3Dmedia%26token%3D300e38ad-6415-44b4-b44e-095012e3fb9c&w=256&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </span>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
